modules/ROOT/pages/installation/linux/tarball.adoc

```diff
@@ -87,8 +87,13 @@ TimeoutSec=120
 
 [Install]
 WantedBy=multi-user.target
-//Reload systemctl to pick up the new service file
-systemctl daemon-reload
+----
+
+. Reload systemctl to pick up the new service file:
++
+[source, shell]
+----
+systemctl enable neo4j
 ----
 
 . Configure Neo4j to start at boot time:
```

Comment:
It now tells users to run `systemctl enable neo4j` twice in a row, which looks a bit weird.
Could the line `. Reload systemctl to pick up the new service file:` be merged with the one that says `. Configure Neo4j to start at boot time:` to just be:

`. Reload systemctl to pick up the new service file and configure Neo4j to start at boot time:`

then we just need to tell people to run `systemctl enable neo4j` once.

Reply:
Changed it accordingly.